webapp/search.py

```diff
@@ -0,0 +1,45 @@
+from canonicalwebteam.http import CachedSession
```

Comment:
I think you forgot to remove this file. (With the new search module)

Reply:
Thanks! Removed.